### PR TITLE
Mysql seeding

### PR DESCRIPTION
**Pull request by jeff-vincent**
Add example mysql data seeding job.

### CODE REVIEW

File: examples/aws-mysql-data-seeding/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+gzip /tmp/example_db.sql
+```
+## copy file from Docker continer to host
+```
+docker cp <containerId>:/tmp/example_db.sql.gz /host/path/example_db.sql.gz
+```
+
+## Upload snapshot to Velocity
+```
+veloctl snapshot put --target mysql-seeding-example-b -f example_db_a.sql.gz --default
```
Comment (uribrecher) [inline examples/aws-mysql-data-seeding/README.md:54]: this line is outdated

File: examples/aws-mysql-data-seeding/mysql-single-job.yaml [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+          env:
+            - name: MYSQL_PASSWORD
+              value: '{velocity.v1:mysql.exports.password}'
+            - name: MYSQL_USER
+              value: '{velocity.v1:mysql.exports.user}'
+            - name: DB_NAME
+              value: '{velocity.v1:mysql.exports.dbname}'
+          command: ["/bin/bash", "-c"]
+          args:
+            - gunzip < /mnt/seeds/example_db.sql.gz | mysql --protocol=TCP --host=mysql-database-example --port=3306 -u$MYSQL_USER -p$MYSQL_PASSWORD $DB_NAME
```
Comment (uribrecher) [inline examples/aws-mysql-data-seeding/mysql-single-job.yaml:124]: nice!!

**Review by uribrecher — CHANGES_REQUESTED** [review]
seeding
This is awesome!!

Can you repeat this example also for GCP?

File: examples/aws-mysql-data-seeding/mysql-single-job.yaml [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+        - env:
+            - name: MYSQL_DATABASE
+              value: example_db
+            - name: MYSQL_PASSWORD
+              value: "{velocity.v1.generate:random}"
+            - name: MYSQL_ROOT_PASSWORD
+              value: admin
+            - name: MYSQL_USER
+              value: user123
+          image: bitnami/mysql:5.7
```
Comment (ilaif) [inline examples/aws-mysql-data-seeding/mysql-single-job.yaml:32]: Let's change to the official mysql image

File: examples/aws-mysql-data-seeding/example_db.sql [hunk 4]
```diff
@@ -0,0 +1,51 @@
+-- MySQL dump 10.13  Distrib 8.0.31, for Linux (aarch64)
+--
+-- Host: localhost    Database: example_db
+-- ------------------------------------------------------
+-- Server version	8.0.31
```
Comment (ilaif) [inline examples/aws-mysql-data-seeding/example_db.sql:5]: Reconcile versions (8.0 vs 5.7)

File: examples/aws-mysql-data-seeding/README.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
+# Create mysql db with data.
```
Comment (ilaif) [inline examples/aws-mysql-data-seeding/README.md:1]: Let's add two different instructions: one for a full example "based on the user's existing database" and a second that just references what's already committed for a quick example of how it works

**Pull request merged**